src/docs/product/data-management-settings/event-grouping/stack-trace-rules.mdx

```diff
@@ -264,3 +311,50 @@ family:native stack.package:**/proprietarymodule.so  max-frames=2
 family:native stack.function:KnownBadFunction1  max-frames=5
 family:native stack.function:KnownBadFunction2  max-frames=5
 ```
+
+### Sentinel & Prefix Frames
+
+<Note>
+
+This feature is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony.
+
+If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.
+
+</Note>
+
+If you have the [Grouping Breakdown feature enabled](/product/data-management-settings/event-grouping/grouping-breakdown#enabling-the-grouping-breakdown-beta) (currently in beta),
+Sentry groups events by identifying the most interesting group of frames it can
+find in a stack trace.
```

Comment:
```suggestion
If you have the [Grouping Breakdown feature enabled](/product/data-management-settings/event-grouping/grouping-breakdown#enabling-the-grouping-breakdown-beta) (currently available only to Early Adopters), Sentry groups events by identifying the most interesting group of frames it can find in a stack trace.
```